### PR TITLE
refactor: Rename imported symbols

### PR DESCRIPTION
**Pull request by ExE-Boss**
I’ve waited with opening this until after <https://github.com/jsdom/webidl2js/pull/166> was done to avoid merge conflicts.

This frees up the `impl` variable for use in <https://github.com/jsdom/webidl2js/pull/158> and also makes it clearer that the current `impl` variable refers to `implSymbol`.

### CODE REVIEW

File: lib/constructs/interface.js [hunk 1]
```diff
@@ -492,8 +492,7 @@ class Interface {
   }
 
   generateRequires() {
-    this.requires.addRaw("impl", "utils.implSymbol");
-    this.requires.addRaw("ctorRegistry", "utils.ctorRegistrySymbol");
+    this.requires.addRaw(`{ implSymbol, ctorRegistrySymbol }`, "utils");
```
Comment (ExE-Boss) [inline lib/constructs/interface.js]: I’ve changed this to use object destructuring to match how alias imports are done in **JSDOM**.

File: README.md [hunk 2]
```diff
@@ -26,8 +26,9 @@ will generate a JavaScript wrapper class file roughly like this:
 
 ```js
 const conversions = require("webidl-conversions");
-const impl = require("./utils.js").implSymbol;
-const ctorRegistry = require("./utils.js").ctorRegistrySymbol;
+const utils = require("./utils.js");
+
+const { implSymbol, ctorRegistrySymbol } = utils;
```
Comment (TimothyGu) [inline README.md]: `utils` doesn't seem to be used anywhere. (This is an example so we don't have to do the exact same as the output.)

```suggestion
const { implSymbol, ctorRegistrySymbol } = require("./utils.js");
```

**Comment by TimothyGu** [comment]
Rebased and fixed some issues that came up in rebasing (96794fa).

**Comment by ExE-Boss** [comment]
I had originally fixed that as part of <https://github.com/jsdom/webidl2js/commit/42e8cd58691c203d62ac79b6a39dffd7ff86bac1>.

**Comment by TimothyGu** [comment]
It’s possible that rebasing destroyed the merge. For future could you please rebased rather than merge

**Pull request merged**